app/views/my/dashboard/index.html.slim

```diff
@@ -1 +1,13 @@
-| here in volunteer dashboard...
\ No newline at end of file
+h2 Events
+
+ul.nav.nav-tabs
+  li
+    a New events
+  li
+    a Upcoming events
+  li.active
+    a Past events
+
+= link_to 'New Events', new_events_my_events_path
```

Comment:
move the link to the `New events` tab. Do the same for the upcoming events and past events